app/views/core/debt_too_large.scala.html

```diff
@@ -33,7 +34,7 @@
     loggedIn = signedIn
 ) {
     <h1 class="heading-xlarge">@Messages("ssttp.over-ten-thosand.h1")</h1>
-    <p>@Messages("ssttp.over-ten-thosand.p1")</p>
+    <p>@Messages("ssttp.over-ten-thosand.p1", eligibilityConfig.maximumDebtForSelfServe )</p>
```

Comment:
Trivial: Consider a name change of the message key since it is no longer necessarily a "ten thousand" limit?

Reply:
Hmm. 
There's a fair number of other message keys that run into this as well though, `ssttp.over-ten-thosand.p2`, `ssttp.over-ten-thosand.p2.number`, etc.

Reply:
changed over-ten-thosand to debit-too-large

Reply:
going to change it again as I misspelt debt